src/components/container/DailyDataChart/DailyDataChart.css

```diff
@@ -2,13 +2,18 @@
     position: relative;
 }
 
-.mdhui-daily-data-chart .title {
-    padding: 16px;
-    padding-bottom: 0;
-    position: relative;
+.mdhui-daily-data-chart-title {
+    padding: 0 16px;
+    font-size: 1.2em;
```

Comment:
Why the font increase here?  (This will impact all existing charts for people using this component etc.)

Reply:
The `DailyDataChart` changes have been reverted.